pages/common/rsync.md

```diff
@@ -4,21 +4,21 @@
 > Does not allow transfer between two remote hosts
 > Can transfer single files or files matched by pattern
 
-- transfer file from local to remote host
+- transfer file from local to remote host, filename supports regex.
```

Comment:
Is this true? Or is it a glob expansion from the shell that turns into a list of files?